### PR TITLE
bugfix: handle_untracked_process_id

### PR DESCRIPTION
**Pull request by cgx027**
Handle untracked process ID

@iceiilin @pengz1 @WangWinson

### CODE REVIEW

**Comment by iceiilin** [comment]
:+1:

File: test/benchmark/utils/parser.py [hunk 1]
```diff
@@ -526,7 +526,10 @@ def parse_process_list(filename):
 
 # Return a process name from PID per dict lookup
 def get_process_name(process_dict, pid_str):
-    return process_dict[pid_str]
+    if process_dict.has_key(pid_str):
```
Comment (WangWinson) [inline test/benchmark/utils/parser.py]: Well, I know this works. It's better to simply by call `dict.get` method
e.g.
```python
return process_dict.get(pid_str)
```

**Comment by WangWinson** [comment]
still :+1:

**Comment by pengz1** [comment]
+1

**Comment by yyscamper** [comment]
:+1:

**Comment by cgx027** [comment]
Thanks all for the review, however change also included in https://github.com/RackHD/RackHD/pull/308.

Thus, I close this PR.

**Pull request closed without merging**